### PR TITLE
Rename Presto to Trino in product tests

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

File: testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hdp3/apply-hdp3-config.sh [hunk 1]
```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```
Comment (kokosing) [inline testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hdp3/apply-hdp3-config.sh:1]: separate commit? Were these files unsed?

**Review by kokosing — APPROVED** [review]

**Comment by wendigo** [comment]
Requires https://github.com/trinodb/docker-images/pull/85

**Review by losipiuk — APPROVED** [review]
skimmed

**Comment by bitsondatadev** [comment]
:wave: @wendigo - this PR is inactive and doesn't seem to be under development, and it might already be implemented. If you'd like to continue work on this at any point in the future, feel free to re-open.

**Pull request closed without merging**